src/content/docs/en/pages/devtools/cli/azion-cli/commands/build/build.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Azion CLI build
+description: Build an application and prepare it for deployment on the edge.
+meta_tags: cli, next, javascript, astro, vue, react, edge computing
+namespace: documentation_cli_build
+menu_namespace: cliMenuAlpha
+permalink: /documentation/devtools/cli/build/
+---
+
+Build your edge application locally and prepare to deploy it on the edge.
+
+:::note 
+In case one or more required fields aren't informed through the specific flags, an interactive message prompts and asks for the missing information.
```

Comment:
```suggestion
In case one or more required fields aren't informed through the specific flags, an interactive prompt asks for the missing information.
```